packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -2676,9 +2700,17 @@ describe('KeyringController', () => {
 
     it('should unlock also with unsupported keyrings', async () => {
       await withController(
-        { cacheEncryptionKey: true },
+        {
+          cacheEncryptionKey: true,
+          skipVaultCreation: true,
+          state: {
+            vault: JSON.stringify({ data: '0x123', salt: 'my salt' }),
+            // @ts-expect-error we want to force the controller to have an
```

Comment:
Hmm. Strange that the constructor is typed to only allow a subset of state, seems like a mistake in the type definition. This shouldn't be an error.

Ah well, we can fix later.